x/programs/simulator/src/lib.rs

```diff
@@ -16,6 +16,9 @@ mod bindings {
         type Target = [u8];
 
         fn deref(&self) -> &Self::Target {
+            // # Safety:
+            // These bytes were mallocated by CGo
```

Comment:
```suggestion
            // These bytes were allocated by CGo
```
"allocated" is the more generic term